python/ray/monitor.py

```diff
@@ -301,12 +301,16 @@ def local_scheduler_info_handler(self, unused_channel, data):
 
         # Update the load metrics for this local scheduler.
         client_id = binascii.hexlify(message.DbClientId()).decode("utf-8")
+
         ip = self.local_scheduler_id_to_ip_map.get(client_id)
         if ip:
             self.load_metrics.update(ip, static_resources, dynamic_resources)
         else:
             print("Warning: could not find ip for client {}."
                   .format(client_id))
+            for keys, values in self.local_scheduler_id_to_ip_map.items():
+                print(keys)
+                print(values)
```

Comment:
Should this be a single print of the entire map?

Reply:
That is my bad. This was some later debugging I was doing. Didn't know it would automatically end up here. I will remove it.